### PR TITLE
Consistency fixes (across 46 commands), additional CoreProtect integration (/dispfill and /setlever), and a critical bugfix (/setlever) (FS-162)

### PR DESCRIPTION
**Pull request by VideoGameSmash12**
https://forum.totalfreedom.me/d/1301-bug-report-setlever-can-crash-the-server

Make sure to review my changes to make sure I didn't fuck up somewhere down the line.

### Consistency Fixes
I tried to change as many as possible to use `FreedomCommand.msg` instead of `CommandSender.sendMessage` for their messages. Here are a list of the files containing those changes:

* Command_adminworld.java
* Command_adventure.java
* Command_banip.java
* Command_blockedit.java
* Command_blockpvp.java
* Command_cage.java
* Command_cartsit.java
* Command_clearchat.java
* Command_clearinventory.java
* Command_commandlist.java
* Command_creative.java
* Command_deop.java
* Command_deopall.java
* Command_dispfill.java
* Command_doom.java
* Command_gcmd.java
* Command_hubworld.java
* Command_inspect.java
* Command_list.java
* Command_lockup.java
* Command_manageshop.java
* Command_manuallyverify.java
* Command_masterbuilderworld.java
* Command_mbconfig.java
* Command_moblimiter.java
* Command_mp44.java
* Command_mute.java
* Command_nickfilter.java
* Command_op.java
* Command_opall.java
* Command_opme.java
* Command_potion.java (Also corrected the inconsistent "player not found" message's color)
* Command_rank.java
* Command_ride.java
* Command_saconfig.java
* Command_scare.java
* Command_setplayerlimit.java
* Command_settotalvotes.java
* Command_smite.java
* Command_spectator.java
* Command_survival.java
* Command_unblockcmd.java
* Command_uncage.java
* Command_unmute.java
* Command_verifynoadmin.java

### Functionality Additions
Here are some commands I added functionality to:
* Command_dispfill.java: Added some code that hooks into the CoreProtect API to log the items being removed from and added into the dispensers.
* Command_setlever.java: Added some code that hooks into the CoreProtect API to log the levers being interacted with.

### Bugfixes
Here are some bug fixes I did:
* Fixes admin notes not displaying for admins, only admin impostors
* Fixes incorrect rank colors for admin chat messages (Discord -> Minecraft)
* Fixes [Media] being appended into admin chat messages (Discord -> Minecraft) improperly

### Critical Bugfix
Here's a command I fixed a critical bug in:
* Command_setlever.java

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_setlever.java
Comment (Focusvity) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_setlever.java:5]: Unused import, please remove.

**Review by Focusvity — CHANGES_REQUESTED** [review]
Also, pleaser remove `org.bukkit.ChatColor` import from `Command_nickfilter` class as it is also unused.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_setlever.java
Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_setlever.java]: Should be fixed now as of a7529b4.

**Review by Focusvity — APPROVED** [review]

**Comment by Focusvity** [comment]
@AtlasMediaGroup/totalfreedom-minecraft-developers Can one of you take a look - as this doesn't have a ticket, another set of eyes is needed to ensure everything is good.
@Wild1145 Once I have approval from another developer, is this all good to be merged - this doesn't have a ticket.

File: src/main/java/me/totalfreedom/totalfreedommod/discord/DiscordToAdminChatListener.java
Comment (Focusvity) [inline src/main/java/me/totalfreedom/totalfreedommod/discord/DiscordToAdminChatListener.java:88]: Not needed as there is a check on line 91.

**Review by Focusvity — DISMISSED** [review]
Apart from what I have commented, everything looks good.

File: src/main/java/me/totalfreedom/totalfreedommod/discord/DiscordToAdminChatListener.java
Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/discord/DiscordToAdminChatListener.java]: oops, fixed

**Review by Focusvity — APPROVED** [review]
Everything looks good now.
@Wild1145 Is this alright to be merged (no ticket) or do you want me to create a ticket with the changes?
@AtlasMediaGroup/totalfreedom-minecraft-developers Can one of you review this, please?

**Review by Paldiu — APPROVED** [review]

**Comment by Focusvity** [comment]
@Wild1145 I can't trigger Travis-CI to build so can you do it for us, please?

**Comment by Focusvity** [comment]
I have created a ticket (FS-162) and linked the PR to it. It's up to @Wild1145 whether he wants it merged in this update or the next update.

**Comment by Wild1145** [comment]
As I've merged another PR in this should now trigger a new Travis CI Build.

**Pull request merged**
